.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: black
```

Comment:
https://black.readthedocs.io/en/stable/version_control_integration.html

The black docs recommend storing the configuration in a pyproject.toml to ensure editors and direct calls to black behave consistently.

Reply:
Can't +1 this enough. Some folks prefer to run these formatting commands manually from their CLI in addition to all the hooks and whatnot. Consistency across these two invocation styles is important!

I've run into problems with this at work.